### ISSUE TITLE
Add Support for Deconvolution Layer for Keras model

### ISSUE THREAD

**Issue by clover113** [tensorflow/tfjs-converter#67]
Currently, when converting keras model with deconvolution2d layer, the converter went fine. However, when importing the model in Tensorflow.js, it outputs "Error: Unknown layer: Conv2DTranspose". 
Also, conv2dtranspose is supported in tensorflow.js.

**Comment by clover113** [comment]
Wait, conv2dtranspose is not a layer in tensorflow.js. So I think it should be an upstream problem.

**Comment by carlthome** [comment]
Similarly when going through the SavedModel converter with [`tf.layers.conv2d_transpose`](https://www.tensorflow.org/api_docs/python/tf/layers/conv2d_transpose) you'll end up with 
```
Unsupported Ops in the model
StridedSlice
```

**Comment by clover113** [comment]
Yeah, it seems that tensorflow.js's support for layers is too limited. But it is not good not to have Conv2dTransposed layer.

**Comment by nsthorat** [comment]
@caisq can we add this layer next? The op exists in core in forward mode (we'll eventually get to the backprop for it but it's non-trivial).

**Comment by caisq** [comment]
@clover113 Thanks for filing the bug! We can add this layer, but it'll support inference first, without supporting training because the gradients for conv2DTranspose is still missing.

**Comment by dsmilkov** [comment]
This issue was moved to tensorflow/tfjs#53
